graphistry/dgl_utils.py

```diff
@@ -555,156 +531,155 @@ def __getitem__(self, idx):
-    #     use_edge_columns=good_cols_without_label,
-    #     use_node_columns=use_cols,
-    #     use_node_scaler="robust",
-    #     use_edge_scaler="robust",
-    # )
-    # # the DGL graph
-    # G = g2.DGL_graph
-    #
-    # # to get a sense of the different parts in training loop above
-    # # labels = torch.tensor(T.values, dtype=torch.float)
-    # train_mask = G.edata["train_mask"]
-    # test_mask = G.edata["test_mask"]
-    #
-    # # define the model
-    # n_feat = G.ndata["feature"].shape[1]
-    # latent_dim = 32
-    # n_output_feats = (
-    #     16  # this is equal to the latent dim output of the SAGE net, not n_targets
-    # )
-    #
-    # node_features = G.ndata["feature"].float()
-    # edge_label = G.edata["target"]
-    # n_targets = edge_label.shape[1]
-    # labels = edge_label.argmax(1)
-    # train_mask = G.edata["train_mask"]
-    #
-    # # instantiate model
-    # model = LinkPredModelMultiOutput(
-    #     n_feat, latent_dim, n_output_feats, n_targets
-    # )  # 1) #LinkPredModel(n_feat, latent_dim, n_output_feats)
-    #
-    # pred = model(G, node_features)  # the untrained graph
-    #
-    # print(
-    #     f"output of model should have same length as the number of edges: {pred.shape[0]}"
-    # )
-    # print(f"number of edges: {G.num_edges()}")
-    # assert G.num_edges() == pred.shape[0], "something went wrong"
-    #
-    # # the optimizer does all the backprop
-    # opt = torch.optim.Adam(model.parameters())
-    #
-    # def evaluate(model, graph, features, labels, mask):
-    #     model.eval()
-    #     with torch.no_grad():
-    #         logits = model(graph, features)
-    #         logits = logits[mask]
-    #         labels = labels[mask]
-    #         _, indices = torch.max(logits, dim=1)
-    #         correct = torch.sum(indices == labels.argmax(1))
-    #         return correct.item() * 1.0 / len(labels)
-    #
-    # use_cross_entropy_loss = True
-    # # train the model
-    # for epoch in range(2000):
-    #     logits = model(G, node_features)
-    #
-    #     if use_cross_entropy_loss:
-    #         loss = F.cross_entropy(logits[train_mask], edge_label[train_mask])
-    #     else:
-    #         loss = ((logits[train_mask] - edge_label[train_mask]) ** 2).mean()
-    #
-    #     pred = logits.argmax(1)
-    #     acc = sum(pred[test_mask] == labels[test_mask]) / len(pred[test_mask])
-    #
-    #     opt.zero_grad()
-    #     loss.backward()
-    #     opt.step()
-    #     if epoch % 100 == 0:
-    #         print(
-    #             f"epoch: {epoch} --------\nloss: {loss.item():.4f}\n\taccuracy: {acc:.4f}"
-    #         )
-    #
-    # # trained comparison
-    # logits = model(G, node_features)
-    # pred = logits.argmax(1)
-    #
-    # accuracy = sum(pred[test_mask] == labels[test_mask]) / len(
-    #     pred[test_mask]
-    # )  # does pretty well!
-    # print("-" * 60)
-    # print(f"Final Accuracy: {100 * accuracy:.2f}%")
+#     import graphistry
```

Comment:
could we just delete?

or this helps dev somehow, and we might as well uncomment?